css/effects-manager.css

```diff
@@ -0,0 +1,68 @@
+.lwfx-effects-manager {
```

Comment:
most variables (`--var(...)`) are from the Lancer system. This provides the theme support